### ISSUE TITLE
No issues reported on Windows

### ISSUE THREAD

**Issue by rraziel** [palantir/tslint#3687]
### Bug Report

- __TSLint version__: `5.9.1`
- __TypeScript version__: `5.5.1`
- __Running TSLint via__: `npm run tslint` which is `tslint --project ./tsconfig.json --config ./conf/tslint.json '**/*.ts' --exclude '**/node_modules/**'`

#### TypeScript code being linted

Just an example as no tslint errors appear on Windows for me right now (or the Windows CI):

https://github.com/pcxdbg/pcxdbg/blob/master/src/app/windows/output-window/output-window-output.ts#L17

with `tslint.json` configuration:

https://github.com/pcxdbg/pcxdbg/blob/master/conf/tslint.json

#### Actual behavior

No error reported on Windows:

https://ci.appveyor.com/project/rraziel/pcxdbg/build/0.3.0-build.266#L37
Same locally on a Windows 10 PC:

```
$ npm run tslint

> pcxdbg@0.3.0 tslint D:\git\pxd\pcxdbg
> tslint --project ./tsconfig.json --config ./conf/tslint.json '**/*.ts' --exclude '**/node_modules/**'
```

#### Expected behavior

Errors as reported on other OSes:

https://travis-ci.org/pcxdbg/pcxdbg/jobs/335876721#L3270
https://circleci.com/gh/pcxdbg/pcxdbg/258

Note: I seem to remember it worked fine on Windows before, so I expect this is some kind of regression or change

**Comment by janis91** [comment]
@rraziel I think the reported TypeScript version is not available yet ;-)
But nevertheless in our project the same issue occurs. On linux there is no problem. On windows with typescript 2.7.2 and tslint 5.9.1 we have the same actual behavior.

Note: It worked on a previous version of tslint.

**Comment by rraziel** [comment]
Some update.

This command does not work (on Windows only):
```
tslint --project ./tsconfig.json --config ./conf/tslint.json '**/*.ts' --exclude '**/node_modules/**'
```
This one, however, does:
```
tslint --project ./tsconfig.json --config ./conf/tslint.json
```
The `tsconfig.json` is enough to compile already (with `tsc -p`), so being sufficient for tslint as well makes sense.

Still a bit odd that the behavior is different depending on the OS. Maybe those extra arguments should always be ignored and a warning issued when both a project file and "normal" arguments are used?
